docs/source/coordinate.rst

```diff
@@ -0,0 +1,178 @@
+Coordinate
+==========
+
+The scene in manim is made up by 8 x 14 grid. The grid is addressed using a numpy
```

Comment:
"By default, the scene..."

Reply:
I didn't know that. Is the dimension dictated by the quality setting?

Reply:
No, but it's possible to zoom the scene.